pep-9999.rst

```diff
@@ -127,16 +138,17 @@ For example::
     movie: MovieRecord = { "name": "Blade Runner",
                            "year": 1982 }
      
-    movie["year"] = 1985  # Fine, mutate methods added in definition
-    movie["name"] = "Terminator"  # Error, name mutator not declared
+    movie["year"] = 1985  # Fine, mutator methods added in definition
+    movie["name"] = "Terminator"  # Type check error, "name" mutator not declared
 
 
-Type Consistency
-----------------
+Type Consistency Rules
```

Comment:
Copied this heading from https://peps.python.org/pep-0589/ too but happy to change it :)

Reply:
Not my fault that mine is better :P (still, this is a "nit")